src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+const wallRect = wall.getBoundingClientRect();
+const spiderRect = spider.getBoundingClientRect();
+
+const centreX = (wallRect.width - spiderRect.width) / 2;
+const centreY = (wallRect.width - spiderRect.width) / 2;
```

Comment:
The calculation for `centreY` should use `wallRect.height` instead of `wallRect.width` to correctly center the spider vertically within the wall.